mephisto/abstractions/providers/mock/wrap_crowd_source.js

```diff
@@ -45,6 +45,12 @@ function handleSubmitToProvider(task_data) {
   return true;
 }
 
+window.HIDE_SUBMIT_BUTTON = (isHidden) => {
```

Comment:
Can you add this property under the `_MEPHISTO_CONFIG_` variable? We added that to avoid polluting the global namespace over time